src/ert/config/queue_config.py

```diff
@@ -20,11 +19,53 @@
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
```

Comment:
Not sure if now is the time, but we talked about potentially adding the following as GENERIC: 

- MAX_RUNTIME
- MAX_SUBMIT
- NUM_CPU
- MEMORY

You would know best if it's better to just implement everything as-is now, or to prepare the way for doing these things in the future.

Reply:
Thats another PR. I think it should be after we merge `python_jobqueue` to `main` as we in any case need to be backwards compatible.

Reply:
Agree